### ISSUE TITLE
Cannot reliably depend on NOW_GITHUB_* environment variables.

### ISSUE THREAD

**Issue by emmenko** [commercetools/commercetools-docs-kit#349]
Ref #342 

TL;DR: the Github ZEIT Now integration seems to alias the domain when a PR is merged to master. This makes sense on one side, because there are technically no changes, but as soon as we rely on `NOW_GITHUB_*` environment variables, we have a problem.

This is the case for things like rendering the cookie consent scripts.

https://github.com/commercetools/commercetools-docs-kit/blob/40be3d66ada8f38671f7844d284d6afca2c2c0b9/packages/gatsby-theme-docs/gatsby-ssr.js#L16-L18

See:
- https://spectrum.chat/zeit/now/now-for-github-can-we-force-deployment-building-upon-merges-to-master~71a27bb7-1b6a-4fdd-bcc6-8d9f1c39f0c0
- https://github.com/zeit/now/issues/2407

We need to find a different solution for conditionally render elements or not, as we can't rely on those environment variables anymore.
